docker-compose.yml

```diff
@@ -24,9 +24,19 @@ services:
             - '/usr/src/app/node_modules'
         ports:
             - 3008:8080
-        depends_on: 
+        depends_on:
             - unitednepali-server
-    
+
+    unitednepali-documentation:
+        build:
+            context: packages/documentation
+            dockerfile: Dockerfile
+        ports:
+          - 3009:3000
+          - 35729:35729
+        volumes:
+          - './packages/documentation:/documentation'
+
```

Comment:
you should also add a volume mount for `./documentation/node_modules`, this is cleaner than copying over the whole folder as you have in the Dockerfile

Reply:
Are you suggesting
1) remove `COPY . /documentation` from the Dockerfile
2) remove `./package/documentation:/documentation` from the docker-compose.yaml and add `./documentation/node_modules:/documenation/node_moudles` to the docker-compose.yml
3) Just add `./documenation/node_modules:/documenation/node_modules`

Or some other combination of things

Reply:
* remove `COPY . /documentation`
* add `/documentation/node_modules`